src/main/resources/application.properties

```diff
@@ -0,0 +1,5 @@
+spring.mail.host = smtp.gmail.com
```

Comment:
This is good. You can also using a `yml` format for properties. Then they will be written in a tree-like structure which improves readibility and is easier to understand